static/styles/portico/portico.css

```diff
@@ -266,6 +265,11 @@ html {
     background-color: hsl(0, 0%, 100%);
     width: calc(100vw - 300px);
     height: calc(100vh - 59px);
+    overflow: unset;
```

Comment:
Instead of this override, we can just remove `overflow: auto` from `.markdown` static/styles/portico/markdown.css. The only other use of `.markdown` is http://localhost:9991/devtools/ which doesn’t need the `overflow: auto`.

Reply:
Sure, that was one of my major concerns. That's why I didn't change there. I will remove the overflow: auto from .markdown

Reply:
Solved.